.github/workflows/perf-test-eks.yaml

```diff
@@ -28,7 +28,6 @@ jobs:
     env:
       PERF_TEST_NUM_SERVICES: ${{ inputs.num_services || 1000 }}
       PERF_TEST_INSTANCES_PER_SERVICE: ${{ inputs.instances_per_service || 2}}
-      PERF_TEST_MESH_VERSION: ${{ inputs.mesh_version }}
```

Comment:
shouldn't we preserve this to be able to run gh action manually with specific version?

Reply:
That's handled [here](https://github.com/Kong/mesh-perf/pull/179/files/f58e81e06c78639184efe83153fd15d5e21febc7#diff-855248a6edf960f9f6baa4b64cd828f1991cf0e9f3e58040b3413b7c9cc44a45R63)

Reply:
ok, I didn't see it